packages/sanity/src/core/studio/tasks/context/enabled/TasksEnabledProvider.tsx

```diff
@@ -0,0 +1,32 @@
+import {useMemo} from 'react'
+import {useFeatureEnabled} from '../../../../hooks'
+import {useWorkspace} from '../../../workspace'
+import {TasksEnabledContext} from './TasksEnabledContext'
+import {TasksEnabledContextValue} from './types'
+
+interface TaksEnabledProviderProps {
+  children: React.ReactNode
+}
+
+export function TasksEnabledProvider({children}: TaksEnabledProviderProps) {
+  // TODO: Restore this once the feature flag is enabled by the ENTX team, see ENTX-1330.
+  const {enabled: featureEnabled, isLoading} = useFeatureEnabled('studioTasks')
+  const workspace = useWorkspace()
+  const enabled = !!workspace.tasks?.enabled
```

Comment:
It is currently not possible to enable/disable tasks from a plugin config. I know this is not relevant for now – but maybe it is a good idea to make sure that it works already now so that we have that logic in place?

Reply:
How will this need to be configured? Do you have any example?